<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AccountWindow</class>
 <widget class="QMainWindow" name="AccountWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QPushButton" name="new_pb">
        <property name="text">
         <string>New</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="remove_pb">
        <property name="text">
         <string>Remove</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QTreeView" name="userList_treeView">
        <property name="editTriggers">
         <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
        </property>
        <property name="headerHidden">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Server:</string>
        </property>
        <property name="buddy">
         <cstring>server</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="server"/>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Username:</string>
        </property>
        <property name="buddy">
         <cstring>username</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="username"/>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Password:</string>
        </property>
        <property name="buddy">
         <cstring>password</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="password">
        <property name="echoMode">
         <enum>QLineEdit::EchoMode::Password</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>First Name:</string>
        </property>
        <property name="buddy">
         <cstring>firstName</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="firstName"/>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Last Name:</string>
        </property>
        <property name="buddy">
         <cstring>lastName</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lastName"/>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>E-mail:</string>
        </property>
        <property name="buddy">
         <cstring>email</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="email"/>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="activate">
        <property name="text">
         <string>Switch to this account</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
